components/badge/badge.html

```diff
@@ -0,0 +1,113 @@
+                </li>
+            </ul>
+            <div class="social">
+                <a href="https://github.com/tushargoyal812" class="nav-icons anchor-style"><i
+                        class="fab fa-github"></i></a>
+                <a href="https://twitter.com/tushargoyal29" class="nav-icons anchor-style"><i
+                        class="fab fa-twitter-square"></i></a>
+                <a href="https://www.linkedin.com/in/tushargoyal29/" class="nav-icons anchor-style"><i
+                        class="fab fa-linkedin"></i></a>
+            </div>
+        </nav>
+    </header>
+    <div class="main-with-sidebar">
+        <div class="sidebar-wrapper">
+            <aside class="sidebar">
+                <h1>Get started</h1>
+                <hr class="hr-margin">
+                <ul class="sidebar-list">
+                    <li class="sidebar-items"><a class="anchor-style links" href="/documentation/documentation.html">Introduction</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Color</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Typography</a></li>
+                </ul>
+                <h1>Components</h1>
+                <hr class="hr-margin">
+                <ul class="sidebar-list">
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/avatar/avatar.html">Avatar</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Alert</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links page-active" href="">Badge</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Cards</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Image</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Input</a></li>
+                </ul>
+            </aside>
+        </div>
+        <main class="main-section">
+            <h1 class="headings">Badge</h1>
+            <p class="heading-details">
+                Badges are small circles, positioned either at top-right or bottom-right of the parent component. Badge
+                can be used to display numbers, online / offline status, depending on where they are used.
+            </p>
+            <hr class="hr-margin">
+            <h2>Badge on icon</h2>
+            <p class="heading-details">
+                badge on icon can be used for showing notifications on any icon.
+            </p>
+            <section class="badges">
+                <i class="fab fa-spotify badge">
+                    <div class="icon">5</div>
+                </i>
+                <i class="fab fa-facebook-square badge">
+                    <div class="icon">9</div>
```

Comment:
The `div` element is the badge here and `i` element is the actual icon. But classes that we've used here seem to tell the opposite. 

Maybe you can rename class `badge` to `has-badge`, that way with the `i` tag it would read like icon has a badge. And rename class `icon` used in the `div` element to `badge` as the content of `div` is actually the badge.

Reply:
yes you are right i fixed it